### ISSUE TITLE
Photon attempts to rewrite urls before site is accessible.

### ISSUE THREAD

**Issue by austinginder** [Automattic/jetpack#2543]
When I put a new WordPress site live, the first thing I do is activate Jetpack. Jetpack sometimes fails due to recent DNS changes however most times after two or three attempts, Jetpack figures out the new DNS and then connects. If Photon is activated all of my images break as Photon hasn't figured out where my new website is. Two thoughts.

1. Is it possible to improve Photon's ability to find new sites? Maybe extra nameserver/DNS lookups when a high number of 404 errors?
2. Is it possible to delay Photon from redirecting urls until it's sure that Photon can actually pull images from the website? Maybe an automatic 24 hour delay if Photon can't see the website?

**Comment by kraftbj** [comment]
I'm leaning toward a wontfix for the 1st item. Photon is, by design, a very simple system to keep everything quick and snappy and I'm not sure the additional work on the server side would be overall good.

For #2, Photon within Jetpack changes the URL to a Photon server's URL, but doesn't have any type of check—anything like that would slow down page rendering, get caught up in caches, etc. Personally, I think it is out of scope for Jetpack but, in theory, there could be a JS-based check that fires off of onError or something to that effect that would replace broken images with something else (either a default image or whatnot).

tl;dr: 1 vote for `wontfix`.

**Comment by dereksmart** [comment]
I'll second the wontfix vote. as @kraftbj so eloquently explained, these types of enhancements are out of the scope of Jetpack/Photon at this time.
